include/lsst/meas/modelfit/UnitSystem.h

```diff
@@ -24,6 +24,8 @@
 #ifndef LSST_MEAS_MODELFIT_UnitSystem_h_INCLUDED
 #define LSST_MEAS_MODELFIT_UnitSystem_h_INCLUDED
 
+#include <math.h>
```

Comment:
Please move this to UnitSystem.cc, and change it to `#include <cmath>`.  The C standard libraries can be used directly from C++, but the C++ wrappers that remove the ".h" and prepend the "c" are preferred.  It's not guaranteed that the stuff you include will be in `std::` without that.